mne/io/egi/egi.py

```diff
@@ -24,8 +24,8 @@ def _read_header(fid):
     if version > 6 & ~np.bitwise_and(version, 6):
         version = version.byteswap().astype(np.uint32)
     else:
-        ValueError('Watchout. This does not seem to be a simple '
-                   'binary EGI file.')
+        raise ValueError('Watchout. This does not seem to be a simple '
+                         'binary EGI file.')
```

Comment:
nice catch but this should be done is a different PR

Reply:
I wasn't sure, I'll open one right now

Reply:
By any chance, do you know why it has been reported by [LGTM](https://lgtm.com/projects/g/mne-tools/mne-python/rev/pr-ed53039d31b3d79a6e49b8288ed75eefb81e3126) only now?